core/templates/dev/head/components/playlist_tile/CollectionPlaylistTileDirective.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Playlist tile for collections.
+ */
+
+oppia.constant(
+  'COLLECTION_VIEWER_URL', '/collection/<collection_id>');
+oppia.constant(
+  'COLLECTION_EDITOR_URL', '/collection_editor/create/<collection_id>');
+
+oppia.directive('collectionPlaylistTile', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        getCollectionId: '&collectionId',
+        getCollectionTitle: '&collectionTitle',
+        getObjective: '&objective',
+        getNodeCount: '&nodeCount',
+        getLastUpdatedMsec: '&lastUpdatedMsec',
+        getThumbnailIconUrl: '&thumbnailIconUrl',
+        getThumbnailBgColor: '&thumbnailBgColor',
+        isLinkedToEditorPage: '=?isLinkedToEditorPage',
+        getCategory: '&category'
+      },
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/components/playlist_tile/' +
+        'collection_playlist_tile_directive.html'),
+      controller: [
+        '$scope', 'oppiaDatetimeFormatter',
+        'COLLECTION_VIEWER_URL', 'COLLECTION_EDITOR_URL', function($scope,
```

Comment:
Break after '(' and have the line breaks in the deps be at the same locations as those in the stringified list.